### ISSUE TITLE
Gradle is still broken , does not download dependencies and does not compile

### ISSUE THREAD

**Issue by Tsoccerguy3** [AlmuraDev/SGCraft#134]
Some of these links are broken , It a scavenger hunt to locate the dependencies , the build.gradle needs to be fixed . 

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 35s

C:\Users\ASUS\Dev\sgc>gradlew build

> Configure project :
This mapping 'snapshot_20180808' was designed for MC 1.12! Use at your own peril.

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':shadowJar'.
> Could not resolve all dependencies for configuration ':runtimeClasspath'.
   > Could not find deobf.dan200.computercraft:ComputerCraft:1.80pr1-build4.
     Searched in the following locations:
         https://files.minecraftforge.net/maven/deobf/dan200/computercraft/ComputerCraft/1.80pr1-build4/ComputerCraft-1.80pr1-build4.pom
         https://files.minecraftforge.net/maven/deobf/dan200/computercraft/ComputerCraft/1.80pr1-build4/ComputerCraft-1.80pr1-build4.jar
         https://repo.maven.apache.org/maven2/deobf/dan200/computercraft/ComputerCraft/1.80pr1-build4/ComputerCraft-1.80pr1-build4.pom
         https://repo.maven.apache.org/maven2/deobf/dan200/computercraft/ComputerCraft/1.80pr1-build4/ComputerCraft-1.80pr1-build4.jar
         https://libraries.minecraft.net/deobf/dan200/computercraft/ComputerCraft/1.80pr1-build4/ComputerCraft-1.80pr1-build4.pom
         https://libraries.minecraft.net/deobf/dan200/computercraft/ComputerCraft/1.80pr1-build4/ComputerCraft-1.80pr1-build4.jar
         http://files.minecraftforge.net/maven/deobf/dan200/computercraft/ComputerCraft/1.80pr1-build4/ComputerCraft-1.80pr1-build4.pom
         http://files.minecraftforge.net/maven/deobf/dan200/computercraft/ComputerCraft/1.80pr1-build4/ComputerCraft-1.80pr1-build4.jar
         http://maven.cil.li/deobf/dan200/computercraft/ComputerCraft/1.80pr1-build4/ComputerCraft-1.80pr1-build4.pom
         http://maven.cil.li/deobf/dan200/computercraft/ComputerCraft/1.80pr1-build4/ComputerCraft-1.80pr1-build4.jar
         https://oss.sonatype.org/content/groups/public/deobf/dan200/computercraft/ComputerCraft/1.80pr1-build4/ComputerCraft-1.80pr1-build4.pom
         https://oss.sonatype.org/content/groups/public/deobf/dan200/computercraft/ComputerCraft/1.80pr1-build4/ComputerCraft-1.80pr1-build4.jar
         http://maven.ic2.player.to/deobf/dan200/computercraft/ComputerCraft/1.80pr1-build4/ComputerCraft-1.80pr1-build4.pom
         http://maven.ic2.player.to/deobf/dan200/computercraft/ComputerCraft/1.80pr1-build4/ComputerCraft-1.80pr1-build4.jar
         http://repo.spongepowered.org/maven/deobf/dan200/computercraft/ComputerCraft/1.80pr1-build4/ComputerCraft-1.80pr1-build4.pom
         http://repo.spongepowered.org/maven/deobf/dan200/computercraft/ComputerCraft/1.80pr1-build4/ComputerCraft-1.80pr1-build4.jar
         http://cc.crzd.me/maven/deobf/dan200/computercraft/ComputerCraft/1.80pr1-build4/ComputerCraft-1.80pr1-build4.pom
         http://cc.crzd.me/maven/deobf/dan200/computercraft/ComputerCraft/1.80pr1-build4/ComputerCraft-1.80pr1-build4.jar
         file:/C:/Users/ASUS/.gradle/caches/minecraft/deobfedDeps/deobf/dan200/computercraft/ComputerCraft/1.80pr1-build4/ComputerCraft-1.80pr1-build4.pom
         file:/C:/Users/ASUS/.gradle/caches/minecraft/deobfedDeps/deobf/dan200/computercraft/ComputerCraft/1.80pr1-build4/ComputerCraft-1.80pr1-build4.jar
         file:/C:/Users/ASUS/Dev/sgc/.gradle/minecraft/ComputerCraft-1.80pr1-build4.jar
         file:/C:/Users/ASUS/Dev/sgc/.gradle/minecraft/ComputerCraft.jar
     Required by:
         project :
   > Could not find deobf.com.almuradev:malisiscore:1.12.2-6.5.0-SNAPSHOT.
     Searched in the following locations:
         https://files.minecraftforge.net/maven/deobf/com/almuradev/malisiscore/1.12.2-6.5.0-SNAPSHOT/maven-metadata.xml
         https://files.minecraftforge.net/maven/deobf/com/almuradev/malisiscore/1.12.2-6.5.0-SNAPSHOT/malisiscore-1.12.2-6.5.0-SNAPSHOT.pom
         https://files.minecraftforge.net/maven/deobf/com/almuradev/malisiscore/1.12.2-6.5.0-SNAPSHOT/malisiscore-1.12.2-6.5.0-SNAPSHOT.jar
         https://repo.maven.apache.org/maven2/deobf/com/almuradev/malisiscore/1.12.2-6.5.0-SNAPSHOT/maven-metadata.xml
         https://repo.maven.apache.org/maven2/deobf/com/almuradev/malisiscore/1.12.2-6.5.0-SNAPSHOT/malisiscore-1.12.2-6.5.0-SNAPSHOT.pom
         https://repo.maven.apache.org/maven2/deobf/com/almuradev/malisiscore/1.12.2-6.5.0-SNAPSHOT/malisiscore-1.12.2-6.5.0-SNAPSHOT.jar
         https://libraries.minecraft.net/deobf/com/almuradev/malisiscore/1.12.2-6.5.0-SNAPSHOT/maven-metadata.xml
         https://libraries.minecraft.net/deobf/com/almuradev/malisiscore/1.12.2-6.5.0-SNAPSHOT/malisiscore-1.12.2-6.5.0-SNAPSHOT.pom
         https://libraries.minecraft.net/deobf/com/almuradev/malisiscore/1.12.2-6.5.0-SNAPSHOT/malisiscore-1.12.2-6.5.0-SNAPSHOT.jar
         http://files.minecraftforge.net/maven/deobf/com/almuradev/malisiscore/1.12.2-6.5.0-SNAPSHOT/maven-metadata.xml
         http://files.minecraftforge.net/maven/deobf/com/almuradev/malisiscore/1.12.2-6.5.0-SNAPSHOT/malisiscore-1.12.2-6.5.0-SNAPSHOT.pom
         http://files.minecraftforge.net/maven/deobf/com/almuradev/malisiscore/1.12.2-6.5.0-SNAPSHOT/malisiscore-1.12.2-6.5.0-SNAPSHOT.jar
         http://maven.cil.li/deobf/com/almuradev/malisiscore/1.12.2-6.5.0-SNAPSHOT/maven-metadata.xml
         http://maven.cil.li/deobf/com/almuradev/malisiscore/1.12.2-6.5.0-SNAPSHOT/malisiscore-1.12.2-6.5.0-SNAPSHOT.pom
         http://maven.cil.li/deobf/com/almuradev/malisiscore/1.12.2-6.5.0-SNAPSHOT/malisiscore-1.12.2-6.5.0-SNAPSHOT.jar
         https://oss.sonatype.org/content/groups/public/deobf/com/almuradev/malisiscore/1.12.2-6.5.0-SNAPSHOT/maven-metadata.xml
         https://oss.sonatype.org/content/groups/public/deobf/com/almuradev/malisiscore/1.12.2-6.5.0-SNAPSHOT/malisiscore-1.12.2-6.5.0-SNAPSHOT.pom
         https://oss.sonatype.org/content/groups/public/deobf/com/almuradev/malisiscore/1.12.2-6.5.0-SNAPSHOT/malisiscore-1.12.2-6.5.0-SNAPSHOT.jar
         http://maven.ic2.player.to/deobf/com/almuradev/malisiscore/1.12.2-6.5.0-SNAPSHOT/maven-metadata.xml
         http://maven.ic2.player.to/deobf/com/almuradev/malisiscore/1.12.2-6.5.0-SNAPSHOT/malisiscore-1.12.2-6.5.0-SNAPSHOT.pom
         http://maven.ic2.player.to/deobf/com/almuradev/malisiscore/1.12.2-6.5.0-SNAPSHOT/malisiscore-1.12.2-6.5.0-SNAPSHOT.jar
         http://repo.spongepowered.org/maven/deobf/com/almuradev/malisiscore/1.12.2-6.5.0-SNAPSHOT/maven-metadata.xml
         http://repo.spongepowered.org/maven/deobf/com/almuradev/malisiscore/1.12.2-6.5.0-SNAPSHOT/malisiscore-1.12.2-6.5.0-SNAPSHOT.pom
         http://repo.spongepowered.org/maven/deobf/com/almuradev/malisiscore/1.12.2-6.5.0-SNAPSHOT/malisiscore-1.12.2-6.5.0-SNAPSHOT.jar
         http://cc.crzd.me/maven/deobf/com/almuradev/malisiscore/1.12.2-6.5.0-SNAPSHOT/maven-metadata.xml
         http://cc.crzd.me/maven/deobf/com/almuradev/malisiscore/1.12.2-6.5.0-SNAPSHOT/malisiscore-1.12.2-6.5.0-SNAPSHOT.pom
         http://cc.crzd.me/maven/deobf/com/almuradev/malisiscore/1.12.2-6.5.0-SNAPSHOT/malisiscore-1.12.2-6.5.0-SNAPSHOT.jar
         file:/C:/Users/ASUS/.gradle/caches/minecraft/deobfedDeps/deobf/com/almuradev/malisiscore/1.12.2-6.5.0-SNAPSHOT/maven-metadata.xml
         file:/C:/Users/ASUS/.gradle/caches/minecraft/deobfedDeps/deobf/com/almuradev/malisiscore/1.12.2-6.5.0-SNAPSHOT/malisiscore-1.12.2-6.5.0-SNAPSHOT.pom
         file:/C:/Users/ASUS/.gradle/caches/minecraft/deobfedDeps/deobf/com/almuradev/malisiscore/1.12.2-6.5.0-SNAPSHOT/malisiscore-1.12.2-6.5.0-SNAPSHOT.jar
         file:/C:/Users/ASUS/Dev/sgc/.gradle/minecraft/malisiscore-1.12.2-6.5.0-SNAPSHOT.jar
         file:/C:/Users/ASUS/Dev/sgc/.gradle/minecraft/malisiscore.jar

**Comment by Dockter** [comment]
Fixed.

**Comment by dallenwilson** [comment]
My apologies for commenting on a closed issue; I judged it better than opening a new one, as the problem is identical and it was only recently closed.

I'm seeing exactly the same missing dependency errors with everything that was added for cofh testing in commit 07dbfa7.

Using ThermalDynamics as an example, build.gradle has:
`deobfCompile "cofh:ThermalDynamics:1.12.2-2.5.5.21:universal"` which causes it to try to download `http://maven.covers1624.net/deobf/cofh/ThermalDynamics/1.12.2-2.5.5.21/ThermalDynamics-1.12.2-2.5.5.21.jar`

That file does not exist, and in fact the whole `http://maven.covers1624.net/deobf/` does not exist. But there is a deobf jar file at `https://maven.covers1624.net/cofh/ThermalDynamics/1.12.2-2.5.5.21/ThermalDynamics-1.12.2-2.5.5.21-deobf.jar`

Changing the line in build.gradle to `compile "cofh:ThermalDynamics:1.12.2-2.5.5.21:deobf"` resolves the error, downloads the new file and the process continues.

After making the same change in build.gradle for all the new deps, I was able to compile the mod. It appears to load and run fine in minecraft.

I have a fair bit of programming experience, but have never used java or gradle. I don't know what other problems changing from `deobfComplie` to `compile` may cause. I can submit a PR with the changes, if you'd like.
